draft-ietf-httpbis-priority.md

```diff
@@ -278,16 +278,17 @@ For both the Priority header field and the PRIORITY_UPDATE frame, the set of
 priority parameters is encoded as a Structured Fields Dictionary (see
 {{Section 3.2 of STRUCTURED-FIELDS}}).
 
-This document defines the urgency(`u`) and incremental(`i`) parameters. When
-receiving an HTTP request that does not carry these priority parameters, a
+This document defines the urgency(`u`) and incremental(`i`) priority parameters.
+When receiving an HTTP request that does not carry these priority parameters, a
 server SHOULD act as if their default values were specified. Note that handling
 of omitted parameters is different when processing an HTTP response; see
```

Comment:
```suggestion
of omitted priority parameters is different when processing an HTTP response; see
```

Reply:
this was lazy me trying to avoid a merge conflict from another PR. But actually I'll probably forget to apply this instance so the conflict will help.